keg/generator.py

```diff
@@ -26,13 +27,13 @@
 )
 from keg.exceptions import KegError, KegKiwiValidationError
 
+log = logging.getLogger('keg')
 
-def create_image_description(image_source,
-                             recipes_root,
-                             data_roots,
-                             dest_dir,
-                             log,
-                             force=False):
+
+def create_image_description(
+    image_source, recipes_root, data_roots,
```

Comment:
no type hints ?

Reply:
I touched generator only to make it flake clean such that we don't get complains. Let us process all the refactoring for generator.py when we work on that issue #10 ok ?